main/webapp/modules/core/scripts/project/browsing-engine.js

```diff
@@ -234,6 +242,22 @@ BrowsingEngine.prototype.removeFacet = function(facet) {
   }
 };
 
+BrowsingEngine.prototype.hideShowBrowsingFacetSplash = function(clicked) {
+  var hideBrowsingFacetSplash = JSON.parse(Refine.getPreference("ui.browsing.hideBrowsingFacetSplash", false));
```

Comment:
```suggestion
  var hideBrowsingFacetSplash = JSON.parse(Refine.getPreference("ui.browsing.hideFacetSplash", false));
```
No need for the Browsing to be repeated.

Reply:
I created different preferences for the facet splash on the browsing div and the history div. If `hideBrowsingFacetSplash` is changed to `hideFacetSplash` it will cause both divs to be affected. Is that what is needed?